botcity/datapool/datapool.py

```diff
@@ -0,0 +1,180 @@
+
+
+import json
+from dataclasses import dataclass
+
+import requests
+
+from botcity.datapool.entry import DataPoolEntry, StateEnum
+from botcity.maestro import BotMaestroSDK
+
+from .enums import ConsumptionPolicyEnum, TriggerEnum
+
+
+@dataclass
+class DataPool:
+    label: str
+    default_activity: str
+    consumption_policy: str = ConsumptionPolicyEnum.FIFO
+    trigger: str = TriggerEnum.ALWAYS
+    schema: list = None
+    auto_retry: bool = True
+    max_auto_retry: int = 0
+    abort_on_error: bool = True
+    max_errors_before_inactive: int = 0
+    item_max_processing_time: int = 0
+    datapool_id: str = None
+    maestro: BotMaestroSDK = None
+    active: bool = True
+    count_pending: int = 0
+    count_processing: int = 0
+    count_done: int = 0
+    count_error: int = 0
+    count_timeout: int = 0
+    avg_done: int = 0
```

Comment:
This information is not necessary here. Users must use the `summary` method to get the proper values.